.github/workflows/terraform.yml

```diff
@@ -18,6 +18,7 @@ jobs:
       AWS_SECRET_ACCESS_KEY:  ${{ secrets.AWS_SECRET_ACCESS_KEY }}
       TF_VAR_heroku_email: ${{ secrets.HEROKU_EMAIL }}
       TF_VAR_heroku_api_key: ${{ secrets.HEROKU_API_KEY }}
+      RECAPTCHA: ${{ secrets.RECAPTCHA }}
```

Comment:
We will need it on the heroku server as well.

Reply:
> We will need it on the heroku server as well.

That's true but this file doesnt have anything related to captcha, right?
Terraform needs heroku creds because it needs to issue new servers or destroy some other resources.

Reply:
Okay, but how will heroku get the env variables then?

Reply:
It should not affect heroku in any way if we remove this line 21